packages/core/src/interpreter.ts

```diff
@@ -525,6 +525,7 @@ export class Interpreter<
         child.stop();
       }
     });
+    this.children.clear();
```

Comment:
Children, while stopped, were being preserved in this map. So when doing this we could end up with some broken states~
```js
service.start()
service.stop()
service.start()
```